src/components/CreateQuestionView.tsx

```diff
@@ -0,0 +1,87 @@
+import { useAtom } from "jotai"
```

Comment:
Is this how people are supposed to create questions with InteractiveQuestion?
Shouldn't we provide a "works out of the box" component and use it here? Or do we need something custom for shoppy?

Reply:
@npretto we need the custom views for the custom themed buttons ([see the product docs](https://www.notion.so/metabase/Make-low-hanging-fruit-improvements-to-Shoppy-12a69354c9018011be49edf18852c7c0?pvs=4#12a69354c901809f9e28fbce6c44b145)) for Shoppy, yes. Using the default component wouldn't work here as we need to demonstrate customization.

Reply:
We need to re-implement all of it just to change the style/position on the buttons? 🤯

Reply:
@npretto With the current `InteractiveQuestion` API or even with the upcoming `CreateQuestion` API, yeah there is no way around this. We'll be implementing the custom picker components (e.g. `FilterPicker`) and more though, so in this specific case we can't get around implementing a custom view.